src/services/messengerSync/consumer.js

```diff
@@ -8,29 +9,58 @@ const QUEUE_EXTERNAL = Configuration.get('RABBITMQ_MATRIX_QUEUE_EXTERNAL');
 
 let channel;
 
-const validateMessage = (content) => (content.userId && (content.courses || content.teams || content.fullSync));
+const validateMessage = (content) => {
+	if (!content.action) {
+		return false;
+	}
+
+	if (!content.userId && !content.schoolId) {
+		return false;
+	}
+
+	return (content.courses || content.teams || content.fullSync);
+};
+
+const sendToQueue = (message) => {
+	const msgJson = JSON.stringify(message);
+	const msgBuffer = Buffer.from(msgJson);
+	channel.sendToQueue(QUEUE_EXTERNAL, msgBuffer, { persistent: true });
+};
 
 const handleMessage = async (incomingMessage) => {
 	const content = JSON.parse(incomingMessage.content.toString());
 	if (!validateMessage(content)) {
-		logger.warning(`MESSENGER SYNC: invalid message in queue ${QUEUE_INTERNAL}`, incomingMessage);
+		logger.warning(`MESSENGER SYNC: invalid message in queue ${QUEUE_INTERNAL}`, content);
```

Comment:
this logging is very unspecific, log directly in validateMessage 
validation may fail, move to try...catch